typescript/packages/jumble/src/views/CharmList.tsx

```diff
@@ -53,8 +51,7 @@ function CharmPreview({ charm, replicaName }: { charm: Charm; replicaName: strin
       <NavLink to={`/${replicaName}/${charmId(charm)}`}>
         <div>
           <h3 className="text-xl font-semibold text-gray-800 mb-4">
-            {(charm[NAME] || "Unnamed Charm") +
-              ` (#${charmId(charm).slice(-4)})`}
+            {(charm.get()[NAME] || "Unnamed Charm") + ` (#${charmId(charm)!.slice(-4)})`}
```

Comment:
charm.key(NAME)

Reply:
became `charm.get()[NAME]`, could also be `charm.key(NAME).get()`. In this case equivalent.